test/unit/utilsTest.js

```diff
@@ -118,4 +105,59 @@ describe('utils', function () {
       })
     })
   })
+
+  describe('checkPrerequisites', function () {
+    describe('when in npm script', function () {
+      beforeEach(function () {
+        utils.checkPrerequisites(callback)
+      })
+
+      it('yields', function () {
+        expect(callback).to.have.been.calledOnce()
+      })
+    })
+  })
+
+  describe('getCommitMessagesSinceLastRelease', function () {
+    describe('something', function () {
+      beforeEach(function () {
+        utils.getCommitMessagesSinceLastRelease(callback)
+      })
+
+      it('calls exec', function () {
+        expect(childProcess.exec).to.have.been.calledOnce()
+      })
+
+      it('yields', function () {
+        expect(callback).to.have.been.calledOnce()
+      })
+    })
+  })
 })
+
+/* utils.getEmail = callback => {
+  git.raw(['config', '--get', 'user.email'], callback)
+}
+
+utils.setEmail = (email, callback) => {
+  if (!email) git.addConfig('user.email', process.env.GITHUB_EMAIL)
+  callback()
+}
+
+utils.getUser = callback => {
+  git.raw(['config', '--get', 'user.name'], callback)
+}
+
+utils.setUser = (name, callback) => {
+  if (!name) git.addConfig('user.name', process.env.GITHUB_EMAIL)
+  callback()
+}
+
+utils.checkBranch = (releaseBranch, callback) => {
+  git.revparse(['--abbrev-ref', 'HEAD'], (err, branch) => {
+    if (err) return callback(err)
+    const currentBranch = process.env.TRAVIS_BRANCH || process.env.CIRCLE_BRANCH || ('' + branch).replace(/\n/, '')
+    if (releaseBranch !== currentBranch) return callback(`Only releasing on ${releaseBranch}`)
+    callback()
+  })
+} */
```

Comment:
Maybe better to remove these comments and leave empty `describe`s or skipped ones?